.gitignore

```diff
@@ -18,3 +18,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
```

Comment:
I think we want this committed to make sure we have the version we expect of each package in production and not just latest.

Reply:
You are right. There were some issues when I was trying to install the react-notifications and forgot to restore it back. Will do a rebase.

Reply:
Yes, commit this @connectbo

Reply:
Ahh I'm just a few seconds late!